rainbows.rb

```diff
@@ -27,3 +27,12 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+rainbow = [
+  {:red => {:r => 255, :g => 0, :b => 0}}
+  {:orange => {:r => 255, :g => 127, :b => 0}}
+  {:yellow => {:r => 255, :g => 255, :b => 0}}
+  {:green => {:r => 0, :g => 255, :b => 0}}
+  {:blue => {:r => 75, :g => 0, :b => 130}}
+  {:purple => {:r => 143, :g => 0, :b => 255}}
+]
```

Comment:
This looks good though! The formatting is just tiny bits off in that it's missing the commas separating each element in the array, but the hashes look correct. 

You didn't get to printing all the keys out so we can see all the words, but we'll go over that part in class together. If it doesn't make sense after that, we should go over it 1-1. 

Thanks for trying it and submitting something :)